src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,76 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = "White";
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+    private static final Random RANDOM = new Random();
+    private int randomNumber = RANDOM.nextInt(5);
+    private double figuresSidesNumber = RANDOM.nextInt(100);
+
+    private Square createSquare() {
+        return new Square(COLOR_SUPPLIER.getRandomColor(),
+                figuresSidesNumber);

```

Comment:
Lets generate random number here? in other way you will have figures with same values